src/xrUICore/Windows/UIFrameWindow.cpp

```diff
@@ -303,28 +303,28 @@ void CUIFrameWindow::draw_tile_line(Frect rect, int i, bool b_horz, Fvector2 con
         while (rect.lt.x + EPS_L < rect.rb.x)
         {
             get_points(rect, i, LTp, RBp, LTt, RBt);
-            rect.lt.x = RBp.x;
```

Comment:
This change is not described in the PR. Is this (and similar below) move intentional?